seed.d/rules.json

```diff
@@ -2,29 +2,36 @@
+        "condition":"CPU and memory average usage is between 5% and 20% for 24 hours"
     },
     {
         "rule_id": "cloud_instance_ros_evaluation|INSTANCE_UNDERSIZED",
         "description":"This instance is undersized and optimization is possible.",
-        "reason": "Based on the host metrics set, we have detected that this {system.instance_type} instance on {system.cloud_provider.upper()} is depleting the provided resources. It is leading us to a judgement that it can be undersized. We apply the logic of combination of all 3 scores with the amount of time within 24h your system spends idling, excluding non business hours.",
-        "resolution": "Please pay attention to workloads running inside the RHEL instance, either utilizing your PCP tools or third party application specific tools to verify that the sizing of this ordered instance type from provider is optimized for your workload.",
-        "condition":"CPU Score + MEM Score + I/O Score (combined)  conditioned IDLE < 30% out of 24h,  are exceeding 14/15"
+        "reason": "Based on the host metrics set, we have detected that this {system.instance_type} instance on {system.cloud_provider.upper()} is depleting the provided resources. It is leading us to a judgement that it can be undersized.\nThe specific issues identified by ROS are:\n{system.summary}",
+        "resolution": "Please pay attention to workloads running inside the RHEL instance, either utilizing your PCP tools or third party application specific tools to verify that the sizing of this ordered instance type from provider is optimized for your workload.\nYou may also want to consider changing the instance type from {system.instance_type} to one of the following instance types, which fit the load better: {system.candidate_instance_types}",
+        "condition":"CPU and memory average usage is over 80% for 24 hours"
+    },
+    {
+        "rule_id": "cloud_instance_ros_evaluation|INSTANCE_UNDER_PRESSURE",
+        "description":"This instance is sized properly but it is suffering from pressure.",
+        "reason": "Based on the host metrics set, we have detected that this {system.instance_type} instance on {system.cloud_provider.upper()} is correctly sized according to its utilization but it is experiencing bottlenecks, probably due to peaks.\nThe specific issues identified by ROS are:\n{system.summary}",
+        "resolution": "Please pay attention to the specific pressure conditions (CPU, memory and/or disk I/O) and try to identify the hours where peaks are happening and why. You may use logs, your PCP tools or third party application specific tools (an APM, for instance). \nYou may also want to consider changing the instance type from {system.instance_type} to a larger one in the dimensions where pressure was identified.",
+        "condition":"System is optimized but Kernel PSI is detected"
     },
     {
         "rule_id": "cloud_instance_ros_evaluation|STORAGE_RIGHTSIZING",
```

Comment:
Need to confirm if we do have `Storage Rightsizing` rule key in Advisor engine.

Reply:
As per offline discussion, currently we don't have required details to compute this state. For future work, we are going to keep state details as it is.